tests/integration/issues/list.rs

```diff
@@ -0,0 +1,19 @@
+use crate::integration::{mock_endpoint, register_test, EndpointOptions};
+
+#[test]
+fn command_issues_list_help() {
+    register_test("issues/issues-list-help.trycmd");
+}
+
+#[test]
```

Comment:
Would it be possible to add some "working" tests as well? Maybe with flags behavior that you added as well? :)